tests/components/deconz/test_cover.py

```diff
@@ -71,7 +79,8 @@ async def test_cover(hass):
     assert "cover.window_covering_device" in gateway.deconz_ids
     assert "cover.unsupported_cover" not in gateway.deconz_ids
     assert "cover.deconz_old_brightness_cover" in gateway.deconz_ids
-    assert len(hass.states.async_all()) == 4
+    assert "cover.window_covering_controller" in gateway.deconz_ids
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5535891)